biogrid/libs/components/src/lib/components.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
What's the use case for components.tsx? Is this just an example of how to write a component?

Reply:
Yep, it was auto added but I can make a different component example which can further better outline the components

Reply:
I changed it to be an example header component